com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/adapter/handler/ConfigurationDoneRequestHandler.java

```diff
@@ -118,9 +118,7 @@ private void handleDebugEvent(DebugEvent debugEvent, IDebugSession debugSession,
                 debugEvent.shouldResume = false;
             }
         } else if (event instanceof StepEvent) {
-            ThreadReference stepThread = ((StepEvent) event).thread();
```

Comment:
If do nothing, just remove this condition switch

Reply:
nice.
done.